README.md

```diff
@@ -21,10 +27,14 @@ A list of amazing Newsletters.
 
 ### CSS
 * [CSS Weekly](http://css-weekly.com/)
+* [CSS-Tricks](https://css-tricks.com) signup is at the bottom of the page.
 
 ### Design
 * [Web Design Weekly](https://web-design-weekly.com/)
 * [Product Design Weekly](http://designweekly.atomic.io/). Digital product design news, articles, showcases and resources, delivered straight to your inbox once a week.
+* [Sidebar](http://sidebar.io). Five design-centric posts each day.
+* [Web Desinger News](http://www.webdesignernews.com)
+* [Responsive Design Weekly](http://responsivedesignweekly.com)
```

Comment:
Which link specifically? I have received updates from all three in the past week.